packages/vite/src/node/__tests__/utils.spec.ts

```diff
@@ -1,40 +1,67 @@
-import { injectQuery, isWindows } from '../utils'
+import { injectQuery, isJSRequest, isWindows } from '../utils'
 
-if (isWindows) {
-  // this test will work incorrectly on unix systems
-  test('normalize windows path', () => {
-    expect(injectQuery('C:\\User\\Vite\\Project', 'direct')).toEqual(
-      'C:/User/Vite/Project?direct'
+describe('injectQuery', () => {
+  if (isWindows) {
+    // this test will work incorrectly on unix systems
+    test('normalize windows path', () => {
+      expect(injectQuery('C:\\User\\Vite\\Project', 'direct')).toEqual(
+        'C:/User/Vite/Project?direct'
+      )
+    })
+  }
+
+  test('path with multiple spaces', () => {
+    expect(injectQuery('/usr/vite/path with space', 'direct')).toEqual(
+      '/usr/vite/path with space?direct'
     )
   })
-}
 
-test('path with multiple spaces', () => {
-  expect(injectQuery('/usr/vite/path with space', 'direct')).toEqual(
-    '/usr/vite/path with space?direct'
-  )
-})
+  test('path with multiple % characters', () => {
+    expect(injectQuery('/usr/vite/not%20a%20space', 'direct')).toEqual(
+      '/usr/vite/not%20a%20space?direct'
+    )
+  })
 
-test('path with multiple % characters', () => {
-  expect(injectQuery('/usr/vite/not%20a%20space', 'direct')).toEqual(
-    '/usr/vite/not%20a%20space?direct'
-  )
-})
+  test('path with %25', () => {
+    expect(injectQuery('/usr/vite/%25hello%25', 'direct')).toEqual(
+      '/usr/vite/%25hello%25?direct'
+    )
+  })
 
-test('path with %25', () => {
-  expect(injectQuery('/usr/vite/%25hello%25', 'direct')).toEqual(
-    '/usr/vite/%25hello%25?direct'
-  )
-})
+  test('path with unicode', () => {
+    expect(injectQuery('/usr/vite/東京', 'direct')).toEqual(
+      '/usr/vite/東京?direct'
+    )
+  })
```

Comment:
Would you check the spacing here to avoid adding these lines to the change set?

Reply:
The spacing are coming, cause it's now wrapped inside the `describe('injectQuery', () => {`

If this is not wanted, please confirm with a message